isb_web/main.py

```diff
@@ -52,9 +52,9 @@
     fastapi.staticfiles.StaticFiles(directory=os.path.join(THIS_PATH, "static")),
     name="static",
 )
-templates = fastapi.templating.Jinja2Templates(
-    directory=os.path.join(THIS_PATH, "templates")
-)
+# templates = fastapi.templating.Jinja2Templates(
+#     directory=os.path.join(THIS_PATH, "templates")
+# )
```

Comment:
Had to comment out all the Jinja stuff, due to 

```
isamples_inabox_1  |   File "/usr/local/lib/python3.9/site-packages/starlette/templating.py", line 52, in __init__
isamples_inabox_1  |     assert jinja2 is not None, "jinja2 must be installed to use Jinja2Templates"
isamples_inabox_1  | AssertionError: jinja2 must be installed to use Jinja2Templates
```

I didn't investigate or try to fix yet, as I didn't need it to verify that things came up correctly.

Reply:
Figured this out.  Poetry bug.